Trace analytics update for 3.2 (#10727)
opensearch-project/documentation-website: _observing-your-data/trace/ta-dashboards.md

- The service view table now includes more quick-select icons, allowing you to view correlated traces and logs in their corresponding views with the correct context passed and to view service details in context without leaving the page, as shown in the following image.

  ![Service table quick select icons]({{site.url}}{{site.baseurl}}/images/ta-service-table-icons.png)

## Configurable service map limits
Introduced 3.2  
{: .label .label-purple }

OpenSearch provides default limits for service map rendering. You can increase these limits to render large topologies more completely or lower them to improve client-side performance when focusing on smaller views. This is especially important in environments with many services or dense interconnections, for which the default limits may result in incomplete maps.

### Service map configuration settings

Two **Advanced Settings** control the size and complexity of service maps generated by service map queries:

- `observability:traceAnalyticsServiceMapMaxNodes`: The maximum number of service nodes displayed. Default is 500. 

> - `observability:traceAnalyticsServiceMapMaxEdges`: The maximum number of edges (service-to-service connections) displayed. Default is 1000.

Reviewer: ```suggestion
- `observability:traceAnalyticsServiceMapMaxEdges`: The maximum number of edges (service-to-service connections) displayed. Default is 1,000.
```

To configure these settings in OpenSearch Dashboards, follow these steps:

> 1. On the main menu, select **Management** > **Dashboard Management** > **Advanced Settings**.

Reviewer: ```suggestion
1. From the main menu, select **Management** > **Dashboard Management** > **Advanced Settings**.
```

In OpenSearch 3.1 and earlier, fixed limits (500 nodes and 1000 edges) could produce incomplete visualizations. With configurable limits in 3.2, you can increase the limits to render large topologies more completely, or lower them to improve client-side performance when focusing on smaller views.

### Service map configuration settings

Two **Advanced Settings** control the size and complexity of Service maps generated by service-map queries:

- `observability:traceAnalyticsServiceMapMaxNodes`: Maximum number of service nodes displayed. **Default:** 500  
- `observability:traceAnalyticsServiceMapMaxEdges`: Maximum number of edges (service-to-service connections) displayed. **Default:** 1000

To configure these settings in OpenSearch Dashboards, go to **Management** > **Advanced Settings** and locate the **Trace Analytics** section. Adjust the values according to your environment and performance requirements, then save your changes. Higher values can increase browser memory and CPU usage.

> The following image shows these settings in **Advanced Settings**:

Reviewer 2: ```suggestion
```

> ![Service map configuration settings]({{site.url}}{{site.baseurl}}/images/ta-application-setting-services.png)

Reviewer 2: ```suggestion
```
